src/Modera/DynamicallyConfigurableMJRBundle/Tests/Unit/Contributions/SettingsSectionsProviderTest.php

```diff
@@ -13,7 +15,12 @@ class SettingsSectionsProviderTest extends \PHPUnit_Framework_TestCase
 {
     public function testGetItems()
     {
-        $provider = new SettingsSectionsProvider();
+        $authorizationChecker = \Phake::mock(AuthorizationCheckerInterface::class);
```

Comment:
Let's be on a safe side - add a test if a user doesn't have necessary privilege as well.